ark-grpc/proto/ark/v1/admin.proto

```
@@ -0,0 +1,155 @@
+syntax = "proto3";
+
+package ark.v1;
+
+import "google/api/annotations.proto";
```

Buf (1.55.1)

import "google/api/annotations.proto": file does not exist

(COMPILE)